PyPoE/poe/file/translations.py

```diff
@@ -1585,7 +1595,8 @@ def _add_translation_hashed(self, translation_id, translation):
                 translation.diff(other)
                 print('')'''
 
-                warnings.warn('Duplicate id "%s"' % translation_id, DuplicateIdentifierWarning)
+                warnings.warn('Duplicate id "%s"' %
+                              translation_id, DuplicateIdentifierWarning)
```

Comment:
https://github.com/Project-Path-of-Exile-Wiki/PyPoE/pull/37#discussion_r744102703

Reply:
Fixing